doc/man/dogecoin-cli.1

```diff
@@ -1,9 +1,9 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
```

Comment:
hmm it says do not modify?

Reply:
I believe it means "If you change this, expect it to be overwritten"

Happy that we re-create this with `help2man` befoer release

Reply:
that's ok with me.